contrib/docker/1-start.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+# This file is part of S3QL.
+#
+# Copyright (c) 2008 Nikolaus Rath <Nikolaus@rath.org>
+#
+# This work can be distributed under the terms of the GNU GPLv3.
+
+function debug_log() {
+  if [ "$VERBOSE" == "true" ]; then
+    echo "$1"
+  fi
+}
+
+mount_s3ql_args=()
+
+if [ -n "$LOG_FILE" ]; then
```

Comment:
maybe add validation of `$LOG_FILE` (that it is not `none`/`syslog`)

Reply:
I thought of doing that, wasn't sure to which extend the image should do warnings/verifications?

another one is MAX_CACHE_ENTRIES which should be at least `$(ulimit -n) + 100` according to the docs, or how `ALLOW_OTHER` and `ALLOW_ROOT` aren't actually compatible? (I mean maybe it resolves in an override, but it seems like behaviour that shouldn't be relied on)

I personally would prefer as an image user for it to warn/fail on "nonsensical" settings, so if everyone agrees I'm happy to add some, but not sure to what extent?

Reply:
added for `syslog`, kept `none` out for now, in case your other PR goes in first, in which case the `none` issue goes away altogether

Reply:
We're now much better off! None is supported and works afaict

Reply:
added failure for `S3QL_LOG=syslog` and `S3QL_SYSTEMD=true`, and warning for `S3QL_LOG!=none` (since, technically, there's no reason to actually forbid users from doing that if they really want)